packages/loaders/src/elements/Dots.tsx

```diff
@@ -21,20 +21,19 @@ const COMPONENT_ID = 'loaders.dots';
 
 interface IDotsProps extends React.HTMLAttributes<SVGSVGElement> {
   /**
-   * Size of the loader. Can inherit from `font-size` styling.
+   * Sets the height and width in pixels. Inherits the parent `font-size` by default
    **/
   size?: string | number;
   /**
-   * Color of the loader. Can inherit from `color` styling.
+   * Sets the fill color
```

Comment:
It's helpful engineering documentation to indicate that the fill inherits from font `color` by default.

Reply:
Nice, will amend.